### DISCUSSION TITLE
Having trouble pattern matching in prerender.ignore

### DISCUSSION THREAD

**Discussion by piercemorris** [nitrojs/nitro#2019] [Q&A] [answered]
My goal is to prerender a lot of dynamic pages i.e. `/stores/<storename>` and I have a list of routes, for example:

- `/stores`
- `/stores/<storename>` etc.

And I would like to ignore just the `/stores` route before the prerender. I assumed that the `prerender.ignore` array would accept glob pattern matching so just the `/stores` path would get excluded in the final pre-render stage by using `/stores`. Instead all routes containing `/stores` will get ignored. This behaves like the `/stores/**` pattern instead I just wish for an exact match for the ignore.

Is there anyway around this or is glob pattern matching not supported for the `prerender.ignore` array?

Thanks in advance :)

**Comment by rona-chong** [comment]
It looks like https://github.com/unjs/nitro/pull/1966 was merged in one day after the latest release, which makes me wanna cry, because I also need this functionality but it's not available yet (even tho the latest docs mention it).

I want to ignore the index.html when prerendering, and when I pass in "/" into the array, nitro ends up simply ignoring ALL of my routes, since all routes start with "/". So I need support for something like "^\/$".

**Comment by rona-chong** [comment, marked as answer]
@piercemorris FYI I was able to ignore my index route with a routeRule. For you that might look like ['/stores': { prerender: false }].  https://nitro.unjs.io/guide/routing#route-rules

Good luck.
